VMBackup/main/fsfreezer.py

```diff
@@ -128,7 +129,9 @@ def __init__(self, patching, logger, hutil):
         else:
             self.logger.log("isArm64Machine : " + str(self.isArm64Machine) + " Using x64 safefreeze binary")
             self.safeFreezeFolderPath = "safefreeze/bin/safefreeze"
-
+        
```

Comment:
Please add a log mentioning that the presence of safeFreezeFolder path will be checked.